.env.example

```diff
@@ -1,7 +1,4 @@
-KOVAN_INFURA_URL=
-KOVAN_DEV_PRIVATE_KEY=
-
-MAINNET_INFURA_URL=
-MAINNET_DEV_PRIVATE_KEY=
-
-ETHERSCAN_API_KEY=
\ No newline at end of file
+INFURA_API_KEY=your infura api key
```

Comment:
You should remove these comments or at least add them as actual comments above.
They seem redundant